arrows/ocv/resection_camera.cxx

```diff
@@ -0,0 +1,218 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// \brief OCV resection_camera algorithm implementation
+
+#include "camera_intrinsics.h"
+#include "resection_camera.h"
+
+#include <opencv2/calib3d/calib3d.hpp>
+#include <opencv2/core/eigen.hpp>
+
+using namespace std;
+using namespace cv;
```

Comment:
I'm generally not a fan of wholesale namespace imports. In the case of `cv` in particular, which is already a very short name, is saving four characters worth the possible reduction in readability?